### PR TITLE
Concf 737 | create API for curated content quality

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
ping @vforge @Warkot

**Comment by Warkot** [comment]
You could also use `sendSeflRequest()` (like [here](https://github.com/Wikia/app/blob/CONCF-737/extensions/wikia/CuratedContent/CuratedContentController.class.php#L197)) to get the data, but I actually like the approach you took. I was thinking about doing the same thing while working on #7666.

LGTM :+1:

I'm just confused about using spaces in brackets (`[ 'foo' ]` vs `['bar']`).

File: extensions/wikia/CuratedContent/CuratedContentController.class.php [hunk 1]
```diff
@@ -393,6 +400,102 @@ function getJsonItem( $titleName, $ns, $pageId ) {
...
+
+	private function getCuratedContentQualityForWiki( $wikiID ) {
+		$curatedContent = $this->getCuratedContentForWiki( $wikiID );
+		$tooLongTitleCount = 0;
+		$missingImagesCount = 0;
+		$totalNumberOfItems = 0;
+		foreach ( $curatedContent as $curatedContentModule => $items ) {
+			foreach ( $items as $item ) {
+				if ( $item['type'] == 'category' ) {
+					if ( strlen( $item['label'] ) > 48 ) {
```
Comment (vforge) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php]: Please move magic number to const :dart:

**Comment by bognix** [comment]
@Warkot - these were private methods, I'm not sure sendSelfRequest() will work

**Comment by vforge** [comment]
LGTM

**Pull request merged**